packages/server/src/enums.js

```diff
@@ -0,0 +1,165 @@
+const MARKET_TYPE = {
+  MARKET: 'MARKET',
+  NON_MARKET: 'NON_MARKET',
+};
+
+const START_AT = {
+  QUESTION_CREATION_TIME: 'QUESTION_CREATION_TIME',
+  AGENT_MEASUREMENT_CREATION_TIME: 'AGENT_MEASUREMENT_CREATION_TIME',
+};
+
+const FINAL_COMPARISON_MEASUREMENT = {
+  LAST_OBJECTIVE_MEASUREMENT: 'LAST_OBJECTIVE_MEASUREMENT',
+  LAST_AGGREGATE_MEASUREMENT: 'LAST_AGGREGATE_MEASUREMENT',
+};
+
+const AGENT_TYPE = {
+  USER: 'USER',
+  BOT: 'BOT',
+};
+
+/**
+ * @todo: He is a small issue. If we have "viewers"
+ * @todo: so then why then can add predictions?
+ * @todo: The better to give them another
+ * @todo: group name.
+ *
+ * @todo: But here is other thing. If we have
+ * @todo: "requireVerification" then this is ok.
```

Comment:
The word "VIEWER" no longer makes sense (that was for an older version). Maybe "UNCERTIFIED_FORECASTER" and "CERTIFIED_FORECASTER"?

Reply:
Mmm, I am not sure and this is why. If we have a channel setting "requireVerification" we should save "viewer" group. Either we should remove the new channel setting and add two groups "UNCERTIFIED_FORECASTER" and "CERTIFIED_FORECASTER". But what if we will add another option for the channel? Then according to our logic, we should add new groups and add groups to combine previous logic so we would have 6 groups or more.

Reply:
Sorry, I think I misunderstood the comment before.